app/ante.go

```diff
@@ -49,14 +49,31 @@ func NewAnteHandler(
 	sendblockOptions := osmoante.NewSendBlockOptions(appOpts)
 	sendblockDecorator := osmoante.NewSendBlockDecorator(sendblockOptions)
 	deductFeeDecorator := txfeeskeeper.NewDeductFeeDecorator(*txFeesKeeper, accountKeeper, bankKeeper, nil)
-	// classicSignatureVerificationDecorator is the old flow that's enbedded into the authenticator ante
-	// to enable a circuit breaker
+
+	// classicSignatureVerificationDecorator is the old flow to enable a circuit breaker
 	classicSignatureVerificationDecorator := sdk.ChainAnteDecorators(
+		// We use the old pubkey decorator here to ensure that accounts work as expected,
+		// in SetPubkeyDecorator we set a pubkey in the account store, for authenticators
+		// we avoid this code path completely.
+		ante.NewSetPubKeyDecorator(accountKeeper),
+		ante.NewValidateSigCountDecorator(accountKeeper),
 		ante.NewSigGasConsumeDecorator(accountKeeper, sigGasConsumer),
 		ante.NewSigVerificationDecorator(accountKeeper, signModeHandler),
 		ante.NewIncrementSequenceDecorator(accountKeeper),
 		ibcante.NewRedundantRelayDecorator(channelKeeper),
 	)
+
+	// authenticatorVerificationDecorator is the new authenticator flow that's enbedded into the circuit breaker ante
+	authenticatorVerificationDecorator := sdk.ChainAnteDecorators(
+		authante.NewSetPubKeyDecorator(accountKeeper),
+		ante.NewValidateSigCountDecorator(accountKeeper),
```

Comment:
if we wanted to submit the sigs without requiring 1 signer = 1 signature as we discussed, we could just modify this decorator.

Reply:
Yeah defo, I think we could add valdiatebasic to this flow for sure